wrap.go

```diff
@@ -31,6 +32,17 @@ func Wrapf(err error, format string, a ...any) error {
 	}
 }
 
+// Cause returns the last error in the stack of wrapped errors.
+func Cause(err error) error {
```

Comment:
When wrapping an error, could we include a pointer to the root cause?

Reply:
Why? 😂

Reply:
```suggestion
func Cause(err error) error {
    if wrapped, ok := err.(wrappedError); ok {
        return wrapped.cause
    }
    return err
}
```

Reply:
I don't understand this suggestion. This would only unwrap `wrappedError`. Using `Unwrap()` allows us to traverse any error with `Unwrap() error` defined. See https://pkg.go.dev/errors

Reply:
I was just suggesting an alternate path to eliminate the `for` loop, but it's really just a micro-optimization.

TIMTOWDI, There Is More Than One Way To Do It.  Your way is fine.